module/channel/src/Domain/Query/ExportQueryInterface.php

```diff
@@ -28,5 +28,5 @@ public function findLastExport(ChannelId $channelId): ?\DateTime;
 
     public function getExportIdsByChannelId(ChannelId $channelId): array;
 
-    public function getChannelTypeByExportId(ExportId $exportId): string;
+    public function getChannelTypeByExportId(ExportId $exportId): ?string;
```

Comment:
one of good practices in databse functions is when you go null result name function find, get use when result is always present.

Reply:
I didn't know that, thank you.